content/src/policies/todoApp.POST.todos.rego

```diff
@@ -1,13 +1,25 @@
 package todoApp.POST.todos
 
-import input.user
-import future.keywords.in
-import data.todoApp.common.is_member_of
+# This policy determines whether the user can create todos
 
 default allowed = false
 
+# Only members of the "resource-creators" instance can create a new Todo.
+#
+# Use the "Check" middleware convention
+# The caller will pass in the following resource context:
+# {
+#   "object_type": "resource-creator",
+#   "object_id": "resource-creators",
+#   "relation": "member"
+# }
+
 allowed {
-  allowedGroups := { "editor", "admin" }
-  some group in allowedGroups
-  is_member_of(user, group)
+  ds.check({
+    "object_type": input.resource.object_type,
```

Comment:
If this module is kept around for backward compatibility, why should it need a resource context? The check call can set
```
{
  "object_type": "resource-creator",
  "object_id": "resource-creators",
  "relation": "member",
  "subject_type": "user",
  "subject_id": input.user.id
}
```

Reply:
we discussed and agreed to keep as is